### DISCUSSION TITLE
Multiple feed settings for followed tags

### DISCUSSION THREAD

**Discussion by desdevol** [dailydotdev/daily#989] [Feature Requests]
I might have many tags that I'd like to follow, and it's easy to miss out articles or getting feed getting flooded by one topic over another. For example, nowadays AI & ML is the hot topic, I might see more of those articles than web development, if I follow them both. Currently I don't see there is any other way to mitigate this, other than unfollow and follow back the tags, which is not a desired UX.

Hopefully we can introduce something like multiple feed settings/profile. For example, I can have 3 feed tags settings, one is frontend development (JS, Vue, React), another one is backend development (PHP, Laravel), another one is about AI (machine learning, chatgpt, facial recognition). I can switch between these tabs/settings/profile to check out the articles of the related tags that I'm interested in.

**Comment by idoshamun** [comment]
Hi @desdevol,

Let's keep the discussion here: https://github.com/dailydotdev/daily/issues/336
